src/app/services/auth/account.service.ts

```diff
@@ -0,0 +1,121 @@
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { SessionStorageService } from 'ngx-webstorage';
+import { Observable, Subject } from 'rxjs';
+import { Account } from 'src/model/account.model';
+import { ApiService } from '../api/api.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AccountService {
+
+  private userIdentity: any;
+    private authenticated = false;
+    private authenticationState = new Subject<any>();
```

Comment:
It looks like this file needs to be reformatted to have 2 spaces instead of 4. I don't care which it is, but it should be consistent across files.

Reply:
Fixed